pkg/ffuf/util.go

```diff
@@ -31,11 +31,13 @@ func UniqStringSlice(inslice []string) []string {
 	return ret
 }
 
-//FileExists checks if the filepath exists and is not a directory
+//FileExists checks if the filepath exists and is not a directory.
+//Returns false in case it's not possible to describe the named file.
 func FileExists(path string) bool {
 	md, err := os.Stat(path)
-	if os.IsNotExist(err) {
+	if err != nil {
```

Comment:
In what kind of scenario is this needed?

Reply:
Hi @joohoi,

I documented it in the commit message too.

> panic happened when the provided path was invalid. Example: ".\wordlist.txt:" as the os.Stat call returned an error different than os.ErrNotExist.

This behavior only happens in Windows systems. Here is a reproducer you can use to test it (Windows only):

```go
package main

import (
	"fmt"
	"os"
)

func fileExists(path string) bool {
	md, err := os.Stat(path)
	if os.IsNotExist(err) {
		return false
	}

	return !md.IsDir()
}

func main() {
	fmt.Println(fileExists("/etc/passwd"))
	fmt.Println(fileExists("/thiswillneverexist"))
	fmt.Println(fileExists("/thispanicsinwindows:"))

}
```

And here's the output of running above program in Windows systems:

![image](https://user-images.githubusercontent.com/6826244/96843760-fb3cc300-144e-11eb-8bcc-34a6d40bf520.png)

As you can see, `md.IsDir()` panics because `os.Stat` returned nil for `md` and an error different than `os.ErrNotExist` for `err`.

Please, let me know if you have further questions.